Makefile

```diff
@@ -33,6 +33,8 @@ run: | build run-kernel-gateway
 	@docker run -it --rm \
 		--name $(DASHBOARD_CONTAINER_NAME) \
 		-p 9700:3000 \
+		-p 9701:3001 \
+		-e HTTPS_HOST_PORT=9701 \
```

Comment:
Is there a better way to do this? I think this is used to do the redirect, where it has to be the **external** port rather than the **internal** one.

So a user would specify one port in `config.json` and another port for Docker? What happens when they're not using Docker? The code is still looking for `HTTPS_HOST_PORT`.